terratorch/models/backbones/prithvi_vit.py

```diff
@@ -85,15 +85,29 @@ def _create_prithvi(
     def checkpoint_filter_wrapper_fn(state_dict, model):
         return checkpoint_filter_fn(state_dict, model, pretrained_bands, model_bands)
 
-    model = build_model_with_cfg(
-        TemporalViTEncoder,
-        variant,
-        pretrained,
-        pretrained_filter_fn=checkpoint_filter_wrapper_fn,
-        pretrained_strict=True,
-        encoder_only=encoder_only,
-        **kwargs,
-    )
+    # When the pretrained configuration is not available in HF, we shift to 
+    # pretrained=False
+    try:
+        model = build_model_with_cfg(
+            TemporalViTEncoder,
+            variant,
+            pretrained,
+            pretrained_filter_fn=checkpoint_filter_wrapper_fn,
+            pretrained_strict=True,
+            encoder_only=encoder_only,
+            **kwargs,
+        )
+    except RuntimeError:
+        print(f"No pretrained configuration was found for the model {variant}.")
+        model = build_model_with_cfg(
+            TemporalViTEncoder,
+            variant,
+            False,
```

Comment:
here it seems to be correct, pse correct above @Joao-L-S-Almeida

Reply:
The difference is that the exception imposes `pretrained=False`:
```
    try:
        model = build_model_with_cfg(
            TemporalViTEncoder,
            variant,
            pretrained,
            pretrained_filter_fn=checkpoint_filter_wrapper_fn,
            pretrained_strict=True,
            encoder_only=encoder_only,
            **kwargs,
        )
    except RuntimeError:
        print(f"No pretrained configuration was found for the model {variant}.")
        model = build_model_with_cfg(
            TemporalViTEncoder,
            variant,
            False,
            pretrained_filter_fn=checkpoint_filter_wrapper_fn,
            pretrained_strict=True,
            encoder_only=encoder_only,
            **kwargs,
        )
```

Reply:
The same here (terratorch/models/backbones/prithvi_swin.py):
    # When the pretrained configuration is not available in HF, we shift to 
    # pretrained=False
    try:
        model: MMSegSwinTransformer = build_model_with_cfg(
            MMSegSwinTransformer,
            variant,
            pretrained,
            pretrained_filter_fn=checkpoint_filter_wrapper_fn,
            pretrained_strict=False,
            feature_cfg={"flatten_sequential": True, "out_indices": out_indices},
            **kwargs,
        )
    except RuntimeError:
        print(f"No pretrained configuration was found for the model {variant}.")
        model: MMSegSwinTransformer = build_model_with_cfg(
            MMSegSwinTransformer,
            variant,
            False,
            pretrained_filter_fn=checkpoint_filter_wrapper_fn,
            pretrained_strict=False,
            feature_cfg={"flatten_sequential": True, "out_indices": out_indices},
            **kwargs,
        )

Reply:
Is it answering the question, @romeokienzler ?